docs/docs/release-notes/syntax-changes-0.22.md

```diff
@@ -0,0 +1,110 @@
+---
+layout: doc-page
+title: Syntax Changes in Dotty 0.22
+---
+
+In 2019, we experimented with several syntax changes in Dotty, most notably in the area of
+implicits replacements. In Dotty 0.22, released on Feb 5, 2020, we have settled on what
+we believe will be the definite syntax for Scala 3. Dotty 0.23 will support only this
+syntax. Previous variants will no longer be supported.
+
+Here is a rundown of how previous variants need to be rewritten in the
+new syntax. This will be useful as a migration and learning help for people who have already
+written code in one of the previous versions of Dotty.
+
+## Given Instances
+
+Given instances are always written with `given` and `as`. The previous use of `:` instead of `as` is no longer supported. Examples:
+```scala
+given intOrd as Ordering[Int] { ... }
+given [T] as Ordering[List[T]] { ... }
+given ctx as ExecutionContext = ...
+given Ordering[String] { ... }
+```
+`as` can be omitted if the instance is anonymous and does not have parameters, as in the last definition above.
+
+## Context Parameters
+
+Context parameters are the replacement of Scala 2's implicit parameters. Context parameters and arguments both start with `using`. Examples:
+```scala
+def max[T](x: T, y: T)(using Ordering[T]): T = ...
+given [T](using Ordering[T]) as Ordering[List[T]] { ... }
+
+max(a, b)(using intOrd)
+```
+The previous syntax that uses `given` also for context parameters and arguments is no longer supported.
+
+Context bounds remain supported as a shorthand for one-parameter typeclass constraints. So the two definitions above could also be written as
+```scala
+def max[T: Ordering](x: T, y: T): T = ...
+given [T: Ordering] as Ordering[List[T]] { ... }
+```
+
+## Context Functions
+
+Implicit function types `implicit A => B` have been replaced with context function types, which are written `A ?=> B`. The syntax `(given A) => B` that was used in earlier Dotty versions is no longer supported.
+
+The same change applies to context function values. It's now
+```scala
+(x: A) ?=> E
+```
+instead of `(implicit x: A) => E` or `(given x: A) => E`.
+
+## Given Imports
+
+The syntax of wildcard given import selectors is now `given _`. Examples
+```scala
+import a.{given _}
+import b.{_, given _}
+```
+The previous syntax, which used just `given` without an underscore is no longer supported. The change was made to better align with typed given import selectors such as `given Ordering[T]`,
```

Comment:
> no longer supported

On 0.22.0-RC1, it's still possible to do `import Foo.given` https://scastie.scala-lang.org/SnWyIoLBROW2VI2kuq8kJA

Reply:
Yes, 0.22 RC is maximally permissive. It's on master that the old forms are removed.